README.md

```diff
@@ -53,10 +53,10 @@ Status is a visual interface to make permanent changes on the Blockchain, it han
 
 - **Evangelism**
 Help us spread the word! Tell a friend *right now*, in fact, tell **everyone** - yell from a mountain if you have to, every person counts! If you've got a great story to tell or have some interesting way you've spread the word about Status let us know about it in our [chat](https://join.status.im/chat/public/status)
-
+  
 ## Give me Binaries!
 
-You can get our Beta builds for both Android and iOS on our [website](https://status.im), through our [nightly builds](https://status.im/nightly/), or by [building it yourself](https://status.im/technical/build_status/).
+You can get our Beta builds for both Android and iOS on our [website](https://status.app), through our [nightly builds](https://status.im/nightly/), or by [building it yourself](doc/starting-guide.md).
```

Comment:
yeah that url doesn't exist, for some reason we still use the old website for that :(

Reply:
The old website has been decomissioned:

- [`infra-misc#01c9fafa`](https://github.com/status-im/infra-misc/commit/01c9fafa) - proxy: remove status.im website, add redirect

It will no longer be used.

Reply:
Yeah what I meant was https://status.app/nightly/ points to a 404 and hence I didn't update it in the docs

Reply:
You can just link the CI build: https://ci.infra.status.im/job/status-mobile/job/nightly/

Reply:
I spoke to website team and will update this when a new page is up on status.app website.